MinnPost/object-sync-for-salesforce
# If a fieldmap has allowed record types, use them to constrain the SOQL sent to Salesforce

## What does this Pull Request do?

If a fieldmap has allowed record types, use them to constrain the SOQL query that is used to pull updated records from Salesforce. This should fix #383.

## How do I test this Pull Request?

- I think make fieldmaps that do and that do not use "allowed record types" and make sure it works in both cases.
